### PR TITLE
Trash Piles

### PR DESCRIPTION
**Pull request by Geevies**
* Added various trash piles to maintenance, holding loot for anyone to discover.

Ported from Vorestation.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/structures/trash_pile.dm [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+		/obj/item/rfd/construction,
+		/obj/item/reagent_containers/hypospray/cmo,
+		/obj/item/gun/projectile/improvised_handgun/loaded,
+		/obj/item/gun/launcher/crossbow,
+		/obj/item/gun/launcher/pneumatic
+		)
+
+/obj/structure/trash_pile/Initialize()
+	. = ..()
+	icon_state = pick("pile1","pile2","pilechair","piletable","pilevending","brtrashpile","microwavepile","rackpile","boxfort","trashbag","brokecomp")
```
Comment (fernerr) [inline code/game/objects/structures/trash_pile.dm:29]:
```suggestion
	if(icon_state == initial(icon_state))
		icon_state = pick(icon_states(icon) - icon_state)
```

Comment (fernerr) [reply code/game/objects/structures/trash_pile.dm:29]: Incase if icon state is set to something specific on the map it stays that way.
Also future proofing if more icon states are added or changed.

**Review by fernerr — COMMENTED** [review]
Even if it kind of goes against the spirit of these piles, I feel they ought to be deconstructible somehow. Incase a space gets renovated or changed.
Perhaps if you click a pile on harm intent/(or maybe a matter decompiler) you start picking it apart and after a lengthy timer it goes away generating no useful items, only a bunch of trash?
If someone is hiding in that pile there'd still be a bunch of time to prepare to be found, unlike in a closet.

I also feel they shouldn't be dense, even if it works for some of the sprites, if you get stuck on a small pile of trash on the floor it doesn't feel that good.

File: aurorastation.dme [hunk 2]
```diff
@@ -2182,7 +2183,6 @@
 #include "code\modules\modular_computers\file_system\programs\medical\suit_sensors.dm"
 #include "code\modules\modular_computers\file_system\programs\pai\access_lock.dm"
 #include "code\modules\modular_computers\file_system\programs\pai\directives.dm"
-#include "code\modules\modular_computers\file_system\programs\pai\atmosphere.dm"
```
Comment (JohnWildkins) [inline aurorastation.dme:2185]: i, uh, don't think you meant to delete the atmosphere program in this PR

**Review by JohnWildkins — CHANGES_REQUESTED** [review]

**Review by fernerr — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**